rules/windows/process_creation/win_susp_tracker_execution.yml

```diff
@@ -0,0 +1,31 @@
+title: DLL Injection with Tracker.exe
+id: 148431ce-4b70-403d-8525-fcc2993f29ea
+description: This rule detects DLL injection and execution via LOLBAS - Tracker.exe
+author: Avneet Singh '@v3t0_', oscd.community
```

Comment:
could you please apply [this](https://github.com/Neo23x0/sigma/pull/1210/files#r506965174) recommendation here as well?